cha/implementaion/googlesample.tex

```diff
@@ -0,0 +1,36 @@
+\section{Foundation of Our Android App}
+This section describes our implementation and adaptation of a sample app by Google called \textit{UniversalMusicPlayer}\footnote{https://github.com/googlesamples/android-UniversalMusicPlayer} as a foundation for our app.
+We use this sample such that we can focus our project on the device communication and psychoacoustic effects from the beginning, rather than building a generic Android music player with suitable \ac{UI}.
+The sample is a large codebase, as such we do not cover everything, but rather we present the elements and general implementation choices.
+The code provided by Google follows the android development guidelines, as such this section will introduce the basic android building blocks that we use in further development of the application.
+
+These basic building blocks of an Android application are: Activities; services; and intents.
+That is, when disregarding \ac{UI} elements, which are: views and fragments.
```

Comment:
I would consider activities UI elements, they are afterall the UI, a fragment is also strictly speaking called an activity fragment and Activities in the app are placed under UI --> i would simply delete the last line